tools/depth-quality/depth-quality-model.cpp

```diff
@@ -46,8 +46,19 @@ namespace rs2
             {
                 if (valid_config = cfg.can_resolve(_pipe))
                 {
-                    _pipe.start(cfg);
-                    break;
+                    static bool device_is_used = false;
+                    try {
+                        _pipe.start(cfg);
+                        break;
+                    }
+                    catch (...)
+                    {
+                        if (!device_is_used)
```

Comment:
This may mislead as any "pipe.start" failure will be presented as "The device is locked by another app"
Is it possible to isolate only the specific usage by providing "catch (tbd_exception)" ?